wapitiCore/attack/mod_nikto.py

```diff
@@ -105,6 +109,26 @@ async def must_attack(self, request: Request):
 
         return request.url == await self.persister.get_root_url()
 
+    async def is_false_positive(self, evil_request: Request, expected_status_codes: List[int]) -> bool:
+        # Check for false positives by asking an improbable file of the same type at the root of the server
+        # Use a dict to cache requests
+        if evil_request.is_directory:
+            request = Request(f"{evil_request.root}{self.random_string}/")
+        else:
+            request = Request(f"{evil_request.root}{self.random_string}{evil_request.file_ext}")
```

Comment:
What about `evil_request.dir_name` instead of `evil_request.root` ?

Reply:
I thought about that unfortunately it generates a lot of additional requests so it comes with a big performance penalty.

I saw Nikto does apply checks on the root so I kept this behavior.